examples/motoko_examples/basic-dao/src/basic_dao.ts

```diff
@@ -0,0 +1,338 @@
+import {
+  Query,
+  Update,
+  int32,
+  Principal,
+  Result,
+  ic,
+  Opt,
+  UpdateAsync,
+} from 'azle';
+import * as Types from './types';
+
+// we should get the init state when we install the canister
+/* stable */
+let init: Types.BasicDaoStableStorage = {
+  accounts: [],
+  proposals: [],
+  system_params: {
+    transfer_fee: { amount_e8s: 0 },
+    proposal_vote_threshold: { amount_e8s: 0 },
+    proposal_submission_deposit: { amount_e8s: 0 },
+  },
+};
+
+/* stable */ let installed = false;
+/* stable */ let installer: string | null = null;
+/* stable */ let accounts = Types.accounts_fromArray(init.accounts);
+/* stable */ let proposals = Types.proposals_fromArray(init.proposals);
+/* stable */ let next_proposal_id: int32 = 0; // TODO should be nat
+/* stable */ let system_params: Types.SystemParams = init.system_params;
+
+// function heartbeat() {
+//   await execute_accepted_proposals();
+// };
+
+// simulate init function
+// for now as a hack we need to call this function manually
+export function initialize(init: Types.BasicDaoStableStorage) {
```

Comment:
You might (probably will) run into uses using the types like you are using them, `Types.WhateverType`. Support for that can be added later. Right now you need to use the type directly, so just import `import { WhateverType }` instead of storing all exports in a variable.

Basically import and use each type individually.

Reply:
If it does work then that's great, but I doubt it will. Just a warning

Reply:
I will give it a try